src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -289,13 +468,33 @@ bool SetInteriorVehicleDataRequest::AreAllParamsReadOnly(
   return true;
 }
 
+bool CheckReadOnlyParamsForAudio(
+    const smart_objects::SmartObject& module_type_params) {
+  if (module_type_params.keyExists(message_params::kEqualizerSettings)) {
+    const auto& equalizer_settings =
+        module_type_params[message_params::kEqualizerSettings];
+    auto it = equalizer_settings.asArray()->begin();
+    for (; it != equalizer_settings.asArray()->end(); ++it) {
+      if (it->keyExists(message_params::kChannelName)) {
+        LOG4CXX_DEBUG(logger_, " READ ONLY parameter: ");
+        return true;
+      }
+    }
+  }
+  return false;
+}
+
 bool SetInteriorVehicleDataRequest::AreReadOnlyParamsPresent(
     const smart_objects::SmartObject& module_data) {
   LOG4CXX_AUTO_TRACE(logger_);
   const smart_objects::SmartObject& module_type_params =
       ControlData(module_data);
   auto it = module_type_params.map_begin();
-  std::vector<std::string> ro_params = GetModuleReadOnlyParams(ModuleType());
+  const std::string module_type = ModuleType();
+  std::vector<std::string> ro_params = GetModuleReadOnlyParams(module_type);
```

Comment:
it is strange kChannelName is not returned in GetModuleReadOnlyParams()

Reply:
@yang1070 because channelName this is a field in the EqualizerSettings structure  https://github.com/smartdevicelink/sdl_core/blob/feature/rc_new_modules/src/components/interfaces/MOBILE_API.xml#L2907

so we have a separate function that checks read only params for AUDIO module
 https://github.com/smartdevicelink/sdl_core/blob/feature/rc_new_modules/src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc#L471